azure-spring-boot-starters/azure-spring-boot-starter/src/main/resources/ApplicationInsights.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings" schemaVersion="2014-05-30">
     <!-- The key from the portal: -->
-    <InstrumentationKey>${ai.instrumentkey}</InstrumentationKey>
+    <InstrumentationKey></InstrumentationKey>
```

Comment:
Do will still need this `ApplicationInsights.xml` file if the configuration is not used?

Reply:
Yes, will remove this. If no this file, there will be many logs out from ai from previous version. It seems they fixed this defect for now.

Reply:
Just make sure that applicationInsights-core `2.0.1` has this issue, but `2.2.0` fixed.